clients/lsp-nix.el

```diff
@@ -54,6 +54,12 @@
   :type 'string
   :package-version '(lsp-mode . "8.0.0"))
 
+(defcustom lsp-nix-nixd-server-arguments '()
```

Comment:
I think setting it to `nil` is recommended for a later version of Emacs. 🤔

Reply:
> I think setting it to `nil` is recommended for a later version of Emacs. 🤔

Huh. Where could I find info on that?

Reply:
It used to be a compile warning (either Emacs 30 or 31), but it's gone now. 🤔 I guess the Emacs Devel included this but reverted it after a few weeks.